qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/QuteAssert.java

```diff
@@ -113,47 +114,48 @@ public static void testCompletionFor(String value, CompletionItem... expectedIte
 	}
 
 	public static void testCompletionFor(String value, String fileUri, CompletionItem... expectedItems)
-			throws Exception {

```

Comment:
Please don't format the code of QuteAssert (only for your new method). It is hard to review it.